### ISSUE TITLE
Generating XML in JDart

### ISSUE THREAD

**Issue by MarwenSami** [psycopaths/jdart#8]
Hello,

Is it possible to generate an xml version of report ?
trying to add `report.publisher+=,xml` to a jpf file doesn't seem to include JDart output along with JPF output.

Thanks

**Comment by ksluckow** [comment]
No, but it has various other outputs: junit test suite and symbolic summaries (which can be stored in json). In fact, a Google Summer of Code student just developed a json outputter for the entire constraints tree.
Making an XML outputter would be quite easy since the analysis result is already a tree. There are various ways you can do that. One way is to use a JPF publisher which would integrate nicely. Alternatively, just generate the xml after the you obtain the analysis result in the JDart shell.
